### PR TITLE
Adjusting expiration time to be 7 days

### PR DESCRIPTION
**Pull request by emersonmellado**

### CODE REVIEW

File: example/lib/helpers.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ function generateToken(req, GUID, opts) {
   // By default, expire the token after 7 days.
   // NOTE: the value for 'exp' needs to be in seconds since
   // the epoch as per the spec!
-  var expiresDefault = Math.floor(new Date().getTime()/1000) + 7*24*60*60;
+  var expiresDefault = Math.floor(Date.now() + (1000 * 60 * 60 * 24 * 7));
```
Comment (nelsonic) [inline example/lib/helpers.js]: `expiresIn` is defined in _seconds_ not Milliseconds. 
see: https://www.npmjs.com/package/jsonwebtoken
image
and:
image

File: example/server.js [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
+
 http.createServer(function (req, res) {
-  var url = req.url;
-  if( url === '/' || url === '/home' ) { app.home(res);            } // homepage
-  else if( url === '/auth')    { app.handler(req, res);            } // authenticator
-  else if( url === '/private') { app.validate(req, res, app.done); } // private content
-  else if( url === '/logout')  { app.logout(req, res, app.done);   } // end session
-  else if( url === '/exit')    { app.exit(res);                    } // for testing ONLY
-  else                         { app.notFound(res);                } // 404 error
+  var path = url.parse(req.url).pathname;
```
Comment (nelsonic) [inline example/server.js]: good plan. 👍

**Comment by emersonmellado** [comment]
Wow. Thanks @nelsonic. I didn't know about this doc.

Apparently they have a simpler way to set the Expiration by using a 7d notation. Check it out! :)

**Comment by nelsonic** [comment]
Yeah, noted the String `"7d"`  for `expiresIn`.
image

It's worth _noting_ that this is a _convenience_ of the `jsonwebtoken` Node Module and not part of the proposed _standard_. but this code works so your PR is good. 👍 
Thanks again!

**Pull request merged**